flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -372,7 +371,7 @@ public void testInsert() throws Exception {
         assertThat(results.toString())
                 .isEqualTo(
                         "[+I[1, 0, static], +I[1, 1, a], +I[2, 0, static], +I[2, 1, b], +I[3, 0, static], +I[3, 1, c]]");
-        tableEnv.executeSql("insert overwrite dest2 partition (p1,p2) select 1,x,y from src")
+        tableEnv.executeSql("insert overwrite table dest2 partition (p1,p2) select 1,x,y from src")
```

Comment:
Actually, `insert overwrite dest2 partition` is not supported in Hive dialect. We need to change it to `insert overwrite table dest2 partition`.
The origin code will fall back to the parser  implemented in  flink-sql-parser which is not what we expect.